File: src/com/dafttech/terra/game/world/subtiles/SubtileDryGrass.java
package com.dafttech.terra.game.world.subtiles;

import com.badlogic.gdx.graphics.g2d.TextureRegion;
import com.dafttech.terra.game.world.World;
import com.dafttech.terra.resources.Resources;

public class SubtileDryGrass extends SubtileGrass {
    public SubtileDryGrass() {
        super();
    }

    @Override
    public TextureRegion getImage() {
        return Resources.TILES.getImage("mask_grass_dry");
    }

    @Override
    public void spread(World world) {
    }
}

File: terra/src/main/scala/com/dafttech/terra/game/world/subtiles/SubtileDryGrass.scala
package com.dafttech.terra.game.world.subtiles

import com.badlogic.gdx.graphics.g2d.TextureRegion
import com.dafttech.terra.engine.TilePosition
import com.dafttech.terra.resources.Resources
import monix.eval.Task

class SubtileDryGrass extends SubtileGrass {
  override val getImage: Task[TextureRegion] = Resources.TILES.getImageTask("mask_grass_dry")

  override def spread(tilePosition: TilePosition): Unit = ()
}